src/lib/style/main.scss

```diff
@@ -13,7 +13,7 @@
 
 // 2. Vendors
 @use 'vendors/normalize';
-@use 'animate.css/animate.css';
+@use 'animate.css/animate';
```

Comment:
For some reason, the old code didn't work with `sass-loader` (which is used by webpack). The compiled CSS shouldn't be affected by this.

In the portals repo, I've been looking into using `sass-loader` removing the CSS compilation step, so this would allow us to simplify things over there.